### PR TITLE
[DCFS-13] Newsletter Form Validation

### PR DESCRIPTION
**Pull request by kemysola**
## Task: `Form Validation`
- `Changes you made to given task`
- @adaugochi Good morning ma, Kindly review.Thank you.

### CODE REVIEW

**Comment by kemysola** [comment]
@adaugochi Good morning ma,Kindly review.

File: submissions/adekemiAwomolo/dom/newsletter.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+let newsletter = document.getElementById('newsletter');
+newspaper_email = document.getElementById('newsemail');
+emailReg=/^\w+([-]?\w+)*@\w+([-]?\w+)*(\w{2,10})+$/;
+
+function validate_data(field,event){
```
Comment (adaugochi) [inline submissions/adekemiAwomolo/dom/newsletter.js:5]: No need to pass `field` as a parameter. It is only `newsemail` that is using this function

File: submissions/adekemiAwomolo/dom/newsletter.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        field.nextElementSibling.innerHTML = "Enter a valid email address";
+        event.preventDefault();
+        return false;
+    }else{
+        field.nextElementSibling.innerHTML = '';
+        return true;
+    }
+}
+
+function newsValidation(event){
```
Comment (adaugochi) [inline submissions/adekemiAwomolo/dom/newsletter.js:20]: No need for this function as well

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/adekemiAwomolo/htmlForm/signup.html [hunk 3]
```diff
@@ -77,8 +79,11 @@ <h2>Already got an account?<a href="#" id='btn'> Sign in.</a></h2>
 				<li>Sign up for our Newsletter
 					<br>
 					<br>
-					<input id='email' type='signup-1'>
-				    <input type="submit" placeholder="submit">
+					<form id ='newsletter'>
+						<input id='newsemail' type='signup-1'>
+						<a class='error-block'></a>
+						<input type="submit" placeholder="submit">
+						</form>
```
Comment (adaugochi) [inline submissions/adekemiAwomolo/htmlForm/signup.html:86]: Kindly format this block of code properly. check from line 82 - 86

File: submissions/adekemiAwomolo/htmlForm/signup.html [hunk 4]
```diff
@@ -66,6 +66,8 @@ <h2>Already got an account?<a href="#" id='btn'> Sign in.</a></h2>
 				</fieldset>
 		</div>
 	</div>
+	<br>
+	<br>
```
Comment (adaugochi) [inline submissions/adekemiAwomolo/htmlForm/signup.html:70]: Use css to style instead of adding `br`

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**